cmd/deploy.go

```diff
@@ -89,60 +92,107 @@ var deployCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("failed to parse certCsrTemplate: %v", err)
 		}
-		// create directory structure and container per node
+
+		nodesMaxWorkers := maxWorkers
+		linksMaxWorkers := maxWorkers
```

Comment:
does it make sense to set nodes/links max workers to len(nodes/links) in case if max-workers > len(nodes/links)?

Reply:
changed